src/sentry_envelope.c

```diff
@@ -166,6 +167,39 @@ envelope_item_get_filename(const sentry_envelope_item_t *item)
     return sentry_value_as_string(filename);
 }
 
+static int
+envelope_item_get_category(const sentry_envelope_item_t *item)
+{
+    const char *ty = sentry_value_as_string(
+        sentry_value_get_by_key(item->headers, "type"));
+    if (sentry__string_eq(ty, "session")) {
+        return SENTRY_RL_CATEGORY_SESSION;
+    } else if (sentry__string_eq(ty, "transaction")) {
+        return SENTRY_RL_CATEGORY_TRANSACTION;
```

Comment:
True, I also saw this when reviewing this again.
WDYT? should I just remove it? Since we don’t have this concept of transaction in native at all at this point.

Reply:
Leave it like this, if there's no implementation of sessions. I'll likely have the DACI ready by end of week, so we can revisit this once we start to implement APM.